src/screens/AgentReg.js

```diff
@@ -0,0 +1,158 @@
+import { useNavigation } from "@react-navigation/native";
+import React, { useState } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  Pressable,
+  TouchableOpacity,
+} from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+
+import CloseIcon from "../../assets/SVG/closeIcon";
+import ImageUploadAgent from "../components/imageUploadAgent";
+import { COLOURS } from "../constants";
+
+export default function AgentRegScreen({ route }) {
+  const [text, onChangeText] = useState();
+  const [setImageUploaded] = useState(false);
```

Comment:
what are you using this variable for?

Reply:
i did it because brandReg was also following type of code . Now i have updated it